engine/xml-config/src/main/java/org/datacleaner/job/JaxbJobReader.java

```diff
@@ -105,6 +106,8 @@
 
 public class JaxbJobReader implements JobReader<InputStream> {
 
+    public static final String DATACLEANER_JAXB_VARIABLE_PREFIX = "datacleaner.jaxb.variable";
```

Comment:
It seems that everywhere that this constant is referred to, it is also concatenated with a dot. So maybe a slight improvement would be to simply add the dot here and avoid all the extra concatenations :-)